tests/test_commands.py

```diff
@@ -0,0 +1,29 @@
+from pim.commands.init import _defaults, _make_package
+from pim.commands.install import install
+from pim.commands.uninstall import uninstall
+
+from click.testing import CliRunner
+
+def _create_test_package():
+    """Helper function to create a test package"""
```

Comment:
small nit, but i've been putting line breaks after and before the """, even for single lines